cmd/lakectl/cmd/branch_tools.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/treeverse/lakefs/pkg/api"
+	"github.com/treeverse/lakefs/pkg/metastore"
+)
+
+func ExtractRepoAndBranchFromDBName(ctx context.Context, dbName string, client metastore.Client) (string, string, error) {
+	metastoreDB, err := client.GetDatabase(ctx, dbName)
+
+	if err != nil {
+		return "", "", fmt.Errorf("failed to get database on extractBranchFromSchema from '%s': %w", dbName, err)
```

Comment:
```suggestion
		return "", "", fmt.Errorf("\get database on extractBranchFromSchema from '%s': %w", dbName, err)
```
(Go errors describe what happened but do not say it is bad)

Also below.